ee/debug/checkups/download_directory.go

```diff
@@ -0,0 +1,117 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+type DownloadDirectory struct {
+	status  Status
+	summary string
+	files   []string
+}
+
+func (c *DownloadDirectory) Name() string {
+	return "Download directory contents"
+}
+
+func (c *DownloadDirectory) Run(_ context.Context, extraFH io.Writer) error {
+	downloadDir := getDownloadDir()
+	if downloadDir == "" {
+		return fmt.Errorf("no default download directory")
+	}
+
+	err := filepath.Walk(downloadDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
```

Comment:
Prefer the single line -- `if err := ...; err != nil {`

Reply:
removed filepath.Walk